docs/_edot-sdks/java/troubleshooting.md

```diff
@@ -7,4 +7,57 @@ parent: EDOT Java
 
 # Troubleshooting the EDOT Java Agent
 
-TODO
\ No newline at end of file
+The sections below are in the order you should follow, unless you have already identified the section you need.
+
+This guide assumes you have tested the other components in the route from application+agent to Elastic Observability (eg collector or APM server, Elasticsearch, and Kibana) and that the problem has been isolated to the application+agent.
+
+## General
+
+Ensure you have set a service name (eg `-Dotel.service.name=Service1` or environment variable `OTEL_SERVICE_NAME` set to `Service1`) otherwise by default the data (traces, metrics, logs) will be sent to `unknown_service_java` - you may be getting data but it may all be under that service
```

Comment:
[for later improvement] here when we have the configuration part ready, I think it would be simpler to link to it for how to set the config option, we only need to explain the "why we need it".